README.rst

```diff
@@ -40,6 +40,35 @@ The AMD Inference Server is integrated with the following libraries out of the g
 * XModel models with `Vitis AI <https://www.xilinx.com/products/design-tools/vitis/vitis-ai.html>`__ on AMD FPGAs
 * A graph of computation including as pre- and post-processing can be written using `AKS <https://github.com/Xilinx/Vitis-AI/tree/v2.5/src/AKS>`__ on AMD FPGAs for end-to-end inference
 
+Quick Start Inference
+----------------------------------------
+
+Quick start requires AMD EPYC CPU::
+
+  # Step 1: Create the example model repository(requires git-lfs)
+  git lfs clone https://github.com/Xilinx/inference-server.git
```

Comment:
The full inference example in the [docs](https://xilinx.github.io/inference-server/main/quickstart_inference.html#) doesn't clone the repository and just downloads the relevant files (which also avoids the need for git/git-lfs). What do you think about doing that inside `fetch_models.sh`?

Reply:
Well, fetch_models.sh as part of the git repo seems necessary since this file must reside somewhere. It is reasonable to 'git clone' if you are running this quick start example. I think it is fine like this.

Reply:
Yes, this is the question I was raising: whether it would be better to put all the logic inside a bash script or clone the repository and do it that way. What do you think?

Reply:
I think I agree. This has less steps and doesn't require git lfs, which can lead to a mistake.

Reply:
Okay, please make the changes. I'd suggest renaming the bash script as well as it's not just fetching models anymore